autoendpoint/src/headers/vapid.rs

```diff
@@ -75,6 +76,29 @@ impl VapidHeader {
             VapidVersionData::Version2 { .. } => 2,
         }
     }
+
+    pub fn sub(&self) -> Result<String, VapidError> {
+        let data: HashMap<String, Value> = serde_json::from_str(&self.token).map_err(|e| {
+            warn!("🔐 Vapid: {:?}", e);
+            VapidError::SubInvalid
+        })?;
+
+        if let Some(sub_candiate) = data.get("sub") {
+            if let Some(sub) = sub_candiate.as_str() {
+                if !sub.starts_with("mailto:") {
```

Comment:
Apparently `https` is [also valid](https://datatracker.ietf.org/doc/html/rfc8292#section-2.1)?